user_guide_src/source/changelogs/v4.5.6.rst

```diff
@@ -29,6 +29,10 @@ Deprecations
 **********
 Bugs Fixed
 **********
+- **FeatureTestTrait:** Allows `#call` and `#post` to process JSON lists. It failed because `RequestTrait#fetchGlobal` expected the second parameter ($index) to be an associative array.
+
+- **RequestTrait:** Allows to get a value by a numeric key if the data is stored as a list and not an associative array.
```

Comment:
```suggestion
- **RequestTrait:** Fixed a bug where the ``fetchGlobal()`` method did not allow handling data by numeric key when stored as a list.
```

Reply:
Test `testFetchGlobalReturnsWithListValues` delete?

Reply:
No, these are fine, although I have one request.

Please add keys to this test so they're visible right away. This will be more consistent with the real request.

```php
$post = [
    '0' => ['foo' => '0'],
    '1' => ['bar' => '1'],
    '2' => ['baz' => '2'],
];
```

Please make sure these are all strings.